diff --git a/activesupport/lib/active_support/backtrace_cleaner.rb b/activesupport/lib/active_support/backtrace_cleaner.rb
index d06f22a..e161ec4 100644
--- a/activesupport/lib/active_support/backtrace_cleaner.rb
+++ b/activesupport/lib/active_support/backtrace_cleaner.rb
@@ -25,7 +25,7 @@ module ActiveSupport
   # of the backtrace, you can call <tt>BacktraceCleaner#remove_filters!</tt>
   # These two methods will give you a completely untouched backtrace.
   #
-  # Inspired by the Quiet Backtrace gem by Thoughtbot.
+  # Inspired by the Quiet Backtrace gem by thoughtbot.
   class BacktraceCleaner
     def initialize
       @filters, @silencers = [], []
diff --git a/activesupport/lib/active_support/time_with_zone.rb b/activesupport/lib/active_support/time_with_zone.rb
index b0d7f32..412c72d 100644
--- a/activesupport/lib/active_support/time_with_zone.rb
+++ b/activesupport/lib/active_support/time_with_zone.rb
@@ -1,3 +1,4 @@
+require 'active_support/duration'
 require 'active_support/values/time_zone'
 require 'active_support/core_ext/object/acts_like'
 
diff --git a/guides/bug_report_templates/action_controller_master.rb b/guides/bug_report_templates/action_controller_master.rb
index 9be8130..1a5cb4d 100644
--- a/guides/bug_report_templates/action_controller_master.rb
+++ b/guides/bug_report_templates/action_controller_master.rb
@@ -1,15 +1,10 @@
-unless File.exist?('Gemfile')
-  File.write('Gemfile', <<-GEMFILE)
-    source 'https://rubygems.org'
-    gem 'rails', github: 'rails/rails'
-    gem 'arel', github: 'rails/arel'
-  GEMFILE
-
-  system 'bundle'
-end
+require 'bundler/inline'
 
-require 'bundler'
-Bundler.setup(:default)
+gemfile(true) do
+  source 'https://rubygems.org'
+  gem 'rails', github: 'rails/rails'
+  gem 'arel', github: 'rails/arel'
+end
 
 require 'rails'
 require 'action_controller/railtie'
diff --git a/guides/bug_report_templates/active_record_master.rb b/guides/bug_report_templates/active_record_master.rb
index 9557f0b..c69b041 100644
--- a/guides/bug_report_templates/active_record_master.rb
+++ b/guides/bug_report_templates/active_record_master.rb
@@ -1,16 +1,11 @@
-unless File.exist?('Gemfile')
-  File.write('Gemfile', <<-GEMFILE)
-    source 'https://rubygems.org'
-    gem 'rails', github: 'rails/rails'
-    gem 'arel', github: 'rails/arel'
-    gem 'sqlite3'
-  GEMFILE
-
-  system 'bundle'
-end
+require 'bundler/inline'
 
-require 'bundler'
-Bundler.setup(:default)
+gemfile(true) do
+  source 'https://rubygems.org'
+  gem 'rails', github: 'rails/rails'
+  gem 'arel', github: 'rails/arel'
+  gem 'sqlite3'
+end
 
 require 'active_record'
 require 'minitest/autorun'
diff --git a/guides/bug_report_templates/generic_master.rb b/guides/bug_report_templates/generic_master.rb
index d930482..4ed34e1 100644
--- a/guides/bug_report_templates/generic_master.rb
+++ b/guides/bug_report_templates/generic_master.rb
@@ -1,16 +1,11 @@
-unless File.exist?('Gemfile')
-  File.write('Gemfile', <<-GEMFILE)
-    source 'https://rubygems.org'
-    gem 'rails', github: 'rails/rails'
-    gem 'arel', github: 'rails/arel'
-  GEMFILE
+require 'bundler/inline'
 
-  system 'bundle'
+gemfile(true) do
+  source 'https://rubygems.org'
+  gem 'rails', github: 'rails/rails'
+  gem 'arel', github: 'rails/arel'
 end
 
-require 'bundler'
-Bundler.setup(:default)
-
 require 'active_support'
 require 'active_support/core_ext/object/blank'
 require 'minitest/autorun'
